CHANGELOG.md

```diff
@@ -1,6 +1,67 @@
 # Changelog
 
-* ## 6.0.0 (2022-01-20)
+## 6.1.0 (2022-02-08)
+
+### Enhancements
+
+* New APIs to support forthcoming feature flag and experiment functionality.
+  For more information, please see https://docs.bugsnag.com/product/features-experiments
+  [#504](https://github.com/bugsnag/bugsnag-unity/pull/504) [#501](https://github.com/bugsnag/bugsnag-unity/pull/501)
+
+* Update bugsnag-cocoa to v6.16.1
```

Comment:
Do we want to adopt the style used in bugsnag-unreal for this now?

https://github.com/bugsnag/bugsnag-unreal/blob/next/CHANGELOG.md?plain=1#L10-L11